samples/msgext-search-quickstart/js/appPackage/manifest.json

```diff
@@ -56,6 +86,6 @@
     "messageTeamMembers"
   ],
   "validDomains": [
-    "<<valid-domain>>"
+    "localhost:3000"
```

Comment:
Are we not going to add ngrok URL here?

You have removed `"<<valid-domain>>"`

Reply:
changed